lightning_client.go

```diff
@@ -3757,3 +3762,60 @@ func (s *lightningClient) SendCustomMessage(ctx context.Context,
 	_, err := s.client.SendCustomMessage(rpcCtx, rpcReq)
 	return err
 }
+
+// SubscribeCustomMessages subscribes to a stream of custom messages, optionally
+// filtering by peer and message type.
+func (s *lightningClient) SubscribeCustomMessages(ctx context.Context) (
+	chan<- CustomMessage, chan<- error, error) {
+
+	rpcCtx, cancel := context.WithTimeout(ctx, s.timeout)
```

Comment:
Not sure about this timeout context, will this also abort the stream?

Reply:
Yeah indeed, nice catch!